transport/src/main/java/io/netty/channel/nio/NioEventLoop.java

```diff
@@ -113,6 +113,8 @@ public Void run() {
         }
     }
 
+    static final long MAX_SCHEDULED_DAYS = 356 * 3;
```

Comment:
add a comment explaining where this number comes from.

Reply:
will do... It came from testing ;)